lnbits/core/models.py

```diff
@@ -141,6 +141,7 @@ class UpdateUserPassword(BaseModel):
     password: str = Query(default=..., min_length=8, max_length=50)
     password_repeat: str = Query(default=..., min_length=8, max_length=50)
     password_old: Optional[str] = Query(default=None, min_length=8, max_length=50)
+    username: Optional[str] = Query(default=..., min_length=2, max_length=20)
```

Comment:
We should have a new cleaner model instead of `UpdateUserPassword`.

```py
class UpdateSuperuserPassword(BaseModel):
    username: Optional[str] = Query(default=..., min_length=2, max_length=20)
    password: str = Query(default=..., min_length=8, max_length=50)
    password_repeat: str = Query(default=..., min_length=8, max_length=50)
```